include/ck/tensor_operation/gpu/device/impl/device_grouped_conv_bwd_data_multiple_d_xdl_cshuffle_v1.hpp

```diff
@@ -1076,7 +885,7 @@ struct DeviceGroupedConvBwdDataMultipleD_Xdl_CShuffle_v1
         std::array<index_t, NDimSpatial> input_left_pads_;
         std::array<index_t, NDimSpatial> input_right_pads_;
 
-        const index_t k_batch_;
+        const index_t k_batch_ = 1;
```

Comment:
Isn't it already initialized at line 518?

Reply:
done